api/src/main/java/org/openmrs/module/kenyaemr/metadata/CommonMetadata.java

```diff
@@ -347,6 +353,8 @@ public void install() {
 		install(form("Diabetic Clinical Form", "Form for Diabetic Consultattion clinical encounter", _EncounterType.DIABETICCONSULTATION, "1", 
 				_Form.DIABETIC_CLINICAL_FORM));
 		install(form("Adverse Drug Reaction Clinical Form", "Form for Adverse Drug Reaction and Pharmacovigilance clinical encounter", _EncounterType.ADVERSEDRUGREACTION, "1", _Form.ADVERSE_DRUG_REACTION_FORM));
+		install(form("Dermatology Clinical Encounter Form", "Form for Dermatology clinical encounter", _EncounterType.DERMATOLOGY, "1", _Form.DERMATOLOGY_CLINICAL_ENCOUNTER_FORM));
+		install(form("Urology Clinical Encounter Form", "Form for Urology clinical encounter", _EncounterType.UROLOGY, "1", _Form.UROLOGY_CLINICAL_ENCOUNTER_FORM));
```

Comment:
```suggestion
		install(form("Dermatology Clinical Form", "Form for Dermatology clinical encounter", _EncounterType.DERMATOLOGY, "1", _Form.DERMATOLOGY_CLINICAL_ENCOUNTER_FORM));
		install(form("Urology Clinical Form", "Form for Urology clinical encounter", _EncounterType.UROLOGY, "1", _Form.UROLOGY_CLINICAL_ENCOUNTER_FORM));
```

Reply:
@ckote  I think we can just omit the word Encounter from the form name what do you think ?